useScroll/Scrollbar.js

```diff
@@ -148,19 +160,7 @@ ScrollbarBase.defaultProps = {
  * @ui
  * @private
  */
-/* TODO: Is it possible to use ApiDecorator?
-const Scrollbar = ApiDecorator(
-	{api: [
-		'focusOnButton',
-		'getContainerRef',
-		'isOneOfScrollButtonsFocused',
-		'showThumb',
-		'startHidingThumb',
-		'update'
-	]}, Skinnable(ScrollbarBase)
-);
-*/
-const Scrollbar = ScrollbarBase;
+const Scrollbar = Skinnable(ScrollbarBase);
 
 Scrollbar.displayName = 'Scrollbar';
 
```

Comment:
We could export the `ScrolbarBase` itself instead `Scrollbar as ScrollbarBase`.

```
export {
	Scrollbar,
	ScrollbarBase
};
```

Reply:
good catch